### ISSUE TITLE
Is there a way to trigger manually?

### ISSUE THREAD

**Issue by huyz** [tth05/obsidian-completr#37]
I may be confused about how to use this.  If I set the "Minimum word trigger length" to say 3, but then I only type 2 letters and hit the hotkey for "Show suggestions", the popup doesn't come up. It seems that "Minimum word trigger length" applies both to the automatic trigger and the manual hotkey trigger?

**Comment by tth05** [comment]
Yes that is correct. Probably makes sense to change this behavior.
